src/framework/card/Card.spec.jsx

```diff
@@ -1,12 +1,16 @@
 
+import React from 'react';
 import { TestCaseFactory } from 'react-test-kit';
 import Card from './Card.jsx';
 
+import { Ribbon } from '../framework';
+
 describe('Card', () => {
   const defaultProps = {
     footerRight: 'test',
     imageSrc: './image.jpg',
     title: 'title',
+    children: [<div className="child" />, <div className="child" />],
```

Comment:
I guess children here is not necessary anymore

Reply:
Sounds good! Let's do that.